pyart/util/xsect.py

```diff
@@ -101,6 +102,89 @@ def cross_section_ppi(radar, target_azimuths):
     return radar_rhi
 
 
+def cross_section_rhi(radar, target_elevations):
+    """
+    Extract cross sections from an RHI volume along one or more elevation
+    angles.
+
+    Parameters
+    ----------
+    radar : Radar
+        Radar volume containing RHI sweeps from which azimuthal
+        cross sections will be extracted.
+    target_elevations : list
+        Elevation angles in degrees where cross sections will be taken.
+
+    Returns
+    -------
+    radar_ppi : Radar
+        Radar volume containing PPI sweeps which contain azimuthal
+        cross sections from the original RHI volume.
+
+    """
+
+    # determine which rays from the rhi radar make up the pseudo PPI
+    pppi_rays = []
+    ppi_nsweeps = len(target_elevations)
+    rhi_nsweeps = radar.nsweeps
+
+    for target_elevation in target_elevations:
+        for sweep_slice in radar.iter_slice():
+            sweep_elevations = radar.elevation['data'][sweep_slice]
+            ray_number = np.argmin(np.abs(sweep_elevations - target_elevation))
+            pppi_rays.append(ray_number + sweep_slice.start)
+
+    _range = _copy_dic(radar.range)
```

Comment:
The lines from here on seems to be nearly identical to those in `cross_section_ppi`.  Would it make sense to make these into a private function which both call?